.mergify.yml

```diff
@@ -15,8 +15,11 @@ pull_request_rules:
         type: APPROVE
         message: MickeyMoz 💪
       merge:
-        method: rebase
-        strict: smart
+        # These PRs are just bumping version or updating suffixes,
+        # and very unlikely to break anything. It's not worth the delay
+        # of rebasing them (sometimes multiple times) before merging.
```

Comment:
_(I wish "time" could be a factor here, if the PR failed and will be restarted by someone, the a rebase may be a good idea. Breakage will be much more likely.)_